c-api/dotnet/src/Picky/Picky.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
```

Comment:
It's possible to provide the nuget metadata directly in the project, and add `GeneratePackageOnBuild` - you'll get a nuget package automatically on build.

Reply:
Oh nice! I'll poke you again about that then